src/tests/ftest/dfuse/mu_mount.py

```diff
@@ -111,31 +111,107 @@ def _check_fail(result):
+    def test_dfuse_mu_mount_uns(self):
+        """JIRA ID: DAOS-10859.
+
+        Use Cases:
+            Verify a container created by non-dfuse user, with a UNS path in dfuse, automatically
+                grants container ACLs to the dfuse user.
+            Verify the non-dfuse user and dfuse user can access the container through dfuse.
+            Verify that revoking the container ACLs for only the dfuse user prevents access
+                through dfuse for both the dfuse user and non-dfuse user.
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=vm
+        :avocado: tags=dfuse,dfuse_mu,daos_cmd
+        :avocado: tags=DfuseMUMount,test_dfuse_mu_mount_uns
+        """
+        dfuse_user = getuser()
+
+        self.log.info('Creating a pool and container for dfuse')
+        pool1 = self.get_pool(connect=False)
+        cont1 = self.get_container(pool1, label='root_cont')
+
+        self.log.info('Starting dfuse with the current user in multi-user mode')
+        self.load_dfuse(self.hostlist_clients)
+        self.dfuse.update_params(pool=pool1.identifier, cont=cont1.label.value, multi_user=True)
+        root_dir = self.dfuse.mount_dir.value
         self.dfuse.run()
 
-        # Give root permission to create containers in the pool
-        pool.update_acl(False, entry="A::root@:rw")
+        self.log.info('Creating a second pool')
+        pool2 = self.get_pool(connect=False)
+
+        self.log.info('Giving root permission to create containers in both pools')
+        pool1.update_acl(False, entry="A::root@:rw")
+        pool2.update_acl(False, entry="A::root@:rw")
 
-        self.log.info('Verify UNS sub-container creation as root in multi-user mode succeeds')
         # DaosCommand cannot be used directly because this needs to run remotely
         daos_command = self.get_daos_command()
         daos_path = os.path.join(daos_command.command_path, daos_command.command)
-        cont_path = os.path.join(root_dir, 'sub_cont')
         first_client = NodeSet(self.hostlist_clients[0])
-        command = command_as_user(
-            '{} container create --type POSIX --path {}'.format(daos_path, cont_path), 'root')
-        if not run_remote(self.log, first_client, command).passed:
-            self.fail('Failed to create sub-container as root in multi-user mode')
-
-        # Verify the container is created correctly
-        command = '{} container get-prop --path {}'.format(daos_path, cont_path)
-        if not run_remote(self.log, first_client, command).passed:
-            self.fail('Failed to get sub-container properties in multi-user mode')
-
-        # List dfuse entries
-        command = 'ls -l {}'.format(root_dir)
-        if not run_remote(self.log, self.hostlist_clients, command).passed:
-            self.fail('Failed to {}'.format(command))
+
+        def _verify_uns(pool_label, cont_label):
+            """Helper function for varying pool/container."""
```

Comment:
Do we want the description for the parameters? Also, "varying pool/container" is a bit unclear to me.

Reply:
"varying" as in different pool/container. I.e. if you look at were `_verify_uns` is called below. If I need to push again, I'll make it more clear and add param description, if that's okay with you